lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -24,7 +24,7 @@ def initialize
 
       @excluded_gems = T.let(development_only_dependencies, T::Array[String])
       @included_gems = T.let([], T::Array[String])
-      @excluded_patterns = T.let(["*_test.rb"], T::Array[String])
+      @excluded_patterns = T.let(["**/*_test.rb"], T::Array[String])
```

Comment:
I do like the behaviour with `FNM_PATHNAME` (even though I got it wrong the first time, lol).

Do you think people would default to thinking about it as including path separators?

Reply:
Maybe we're also missing some tests that would have shown the problem earlier?

Reply:
Yeah, I added a test that covers it. It might be worth printing a warning if a pattern doesn't include a matcher for the path.

I created #923 to improve this aspect.